content/matplotlib/concepts/pyplot/terms/bar/bar.md

```diff
@@ -0,0 +1,30 @@
+# .bar()
+
+## Syntax
+A bar chart/bar graph is a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
```

Comment:
This should be part of your introductory paragraph, not part of the Syntax section.